src/pages/Recipes/RecipeCreation/RecipeCreation.jsx

```diff
@@ -13,9 +13,11 @@ const RecipeCreation = ({
   handleEditLinkClick,
   isError,
   isLoading,
+  edition,
```

Comment:
We could set a default here - in that case we won't need to set the prop explicitly to `false` in `RecipeCreationRouter` 😉 

```suggestion
  edition - false,
```